### PR TITLE
Lesson 7

### PR DESCRIPTION
**Pull request by oiskritsky**
Homework done. created a mobile version of the main page, added input fields, a calendar.

### CODE REVIEW

File: Triphouse-project/index.html [hunk 1]
```diff
@@ -11,52 +11,73 @@
...
+              </div>
           </div>
-        </div>
+          <div class="sbar__third">
+            <input type="text" class="Children" placeholder="Adults - Children - Rooms" />
+          </div>
+          
+          <button>Search</button>
+        </form>
+        <form action="" class="form__mobile">
```
Comment (ghost) [inline Triphouse-project/index.html:66]: 2021-04-19_22h29_55
You can add a white background for the form tag, which helps to hide empty space between.

File: Triphouse-project/index.html [hunk 2]
```diff
@@ -11,52 +11,73 @@
...
-              <img src="./img/svg/logo.svg" alt="TripHouse" class="header_logo-pic" />
-            </a>
+      <div class="main__wrapper">
+        <header class="header">
+          <div class="header__wrapper">
+            <div class="header__logo">
+              <a href="/" class="header__logo-link"> </a>
+            </div>
+            <nav class="header__nav">
+              <ul class="header__list">
```
Comment (ghost) [inline Triphouse-project/index.html:21]: 2021-04-19_22h33_15
Align items.

Comment (oiskritsky) [reply Triphouse-project/index.html:21]: aligh-items: start-end; used

File: Triphouse-project/index.html (hunk 1 above)
Comment (oiskritsky) [reply Triphouse-project/index.html:66]: fixed

**Pull request merged**